instrumentation/opentelemetry-instrumentation-elasticsearch/src/opentelemetry/instrumentation/elasticsearch/__init__.py

```diff
@@ -143,13 +146,16 @@ def wrapper(wrapped, _, args, kwargs):
                     span.set_attribute(key, value)
 
             rv = wrapped(*args, **kwargs)
-            if isinstance(rv, dict) and span.is_recording():
-                for member in _ATTRIBUTES_FROM_RESULT:
-                    if member in rv:
-                        span.set_attribute(
-                            "elasticsearch.{0}".format(member),
-                            str(rv[member]),
-                        )
+            response_hook(span, rv)
             return rv
 
     return wrapper
+
+
+def _default_response_hook(span, response):
+    if isinstance(response, dict) and span.is_recording():
+        for member in _ATTRIBUTES_FROM_RESULT:
+            if member in response:
+                span.set_attribute(
+                    "elasticsearch.{0}".format(member), str(response[member]),
+                )
```

Comment:
Any reason to move logic into a "default hook"? Hooks give users the ability to perform additional operations on or with the span data. There shouldn't be any default hooks containing any logic that gets replaced by a user provided hook. Please move this back to the previous place.